test/mako/alerter/shared/error.go

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 // error.go helps with error handling
 
-package alerter
+package shared
```

Comment:
that's a bad package name... Do we even need it? May be just  keep eveything in alerter?

Reply:
I'll get `import cycle not allowed` error if moving them to `alerter`. Will `common` or `utils` be a better name?

Reply:
move it to `https://github.com/knative/pkg/tree/master/test/helpers` instead? as this not specific to the alerter and can be potentially used by others

Reply:
Done.